### PR TITLE
Daniel/generate events instructions

### PR DESCRIPTION
**Pull request by danielgehrig18**
- simplified installation
- example code to convert video to events.

### CODE REVIEW

File: esim_torch/generate_events.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            process_dir(output_folder, path, args)
+
+
+
+
+
+
+
+    
+
```
Comment (magehrig) [inline esim_torch/generate_events.py:85]: rm whitespace

File: esim_torch/generate_events.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser("""Generate events from a high frequency video stream""")
+    parser.add_argument("--contrast_threshold_negative", "-cn", type=float, default=0.2)
+    parser.add_argument("--contrast_threshold_positive", "-cp", type=float, default=0.2)
+    parser.add_argument("--refractory_period_ns", "-rp", type=int, default=0)
+    parser.add_argument("--input_dir", "-i", default="", required=True)
+    parser.add_argument("--output_dir", "-o", default="", required=True)
```
Comment (magehrig) [inline esim_torch/generate_events.py:66]: I think positional arguments would be better. e.g. 
`parser.add_argument("input_dir")`

File: upsampling/example/.gitignore [hunk 3]
```diff
@@ -1,3 +0,0 @@
-*
```
Comment (magehrig) [inline upsampling/example/.gitignore:1]: why remove this?

File: README.md [hunk 4]
```diff
@@ -37,15 +37,10 @@ git clone git@github.com:uzh-rpg/rpg_vid2e.git --recursive
...
-Adapt the CUDA toolkit version according to your setup. 
 
 ```bash
-cuda_version=10.1
-
-conda create -y -n vid2e python=3.7
-conda activate vid2e
-conda install -y pytorch torchvision cudatoolkit=$cuda_version -c pytorch
-conda install -y -c conda-forge opencv tqdm scikit-video eigen boost boost-cpp pybind11 matplotlib
+conda create --name vid2e --file requirements.txt
```
Comment (magehrig) [inline README.md:42]: best to test that this works without problems. I often run into freezes with requirement/env files.

**Review by magehrig — APPROVED** [review]
see comments otherwise lgtm

File: esim_torch/generate_events.py (hunk 2 above)
Comment (danielgehrig18) [reply esim_torch/generate_events.py:62]: I tried to be consistent with the upsampling code, so left it

File: upsampling/example/.gitignore (hunk 3 above)
Comment (danielgehrig18) [reply upsampling/example/.gitignore:1]: this folder is no longer needed

**Pull request merged**